app/src/main/java/org/schabi/newpipe/util/JSONSerializer.java

```diff
@@ -0,0 +1,37 @@
+package org.schabi.newpipe.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+
+/**
+ * Centralized class for JSON serialization.
+ */
+public final class JSONSerializer {
+    private JSONSerializer() {
+
+    }
+
+    private static ObjectMapper newMapper() {
+        return new ObjectMapper();
+    }
```

Comment:
According to Phind an `ObjectMapper`should be reused, because it's creation is relativly expensive.

SO source: https://stackoverflow.com/a/57671444

It kind of makes sense to me because in the spring boot world you have one instance of it and use it for all http requests unless specific config is required